kivymd/uix/tab.py

```diff
@@ -1079,12 +1172,25 @@ class MDTabs(ThemableBehavior, SpecificBackgroundColorBehavior, AnchorLayout):
 
     allow_stretch = BooleanProperty(True)
     """
-    If False - tabs will not stretch to full screen.
+    If `True`, The tab will update dynamically to it's content width, and wrap
+    any text if the widget is wider than `"360dp"`.
+
+    If `False`, the tab won't update to it's maximum texture width.
+    this means that the `fixed_tab_label_width` will be used as the label
+    width. this will wrap any text inside to fit the fixed value.
 
     :attr:`allow_stretch` is an :class:`~kivy.properties.BooleanProperty`
     and defaults to `True`.
     """
 
+    fixed_tab_label_width = NumericProperty("140dp")
```

Comment:
See the general comment about the change in the meaning of `allow_stetch`

In addition - although this mechanism works and is broadly in line with material.io - I do miss the functionality to ask kivyMD to expand the Tabs to fill the screen. It was useful functionality and was also in line with material.io

Reply:
It is a useful attribute, but, that shouldn't be it's name, since hat you're requiring, it's size_hint_x, from the grid layout that contains all tabs, implementing something to do the same in cpython to normal python is a really bad practice, but, i could forward the size hint of the tab bar to the MDTabsBase. So you could still use something like.
```py
tab_hint_x=1
tab_hint_x=None
```

Reply:
1. It is not a matter of whether you think the name is a good name or whether I think it is a good name - IT IS THE EXISTING API. That is what existing application - some of them things that people make money out of - have included in their applications. Not because they are stupid or bad programmers but because that is what the previous specification WAS.

2. I did not suggest anything about implementation - so I am not sure why you seem to think that I was proposing to change something from cython to python? 

3. I am not sure you understand the proposed solution - not enough info - but if it involves adding NEW complexity to an already complex API then probably not.

I think what I would really would have liked is:

- `allow_stretch` doing exactly what it did before (because :: legacy) and deprecated (because :: not really MD and badly named), and

- A "tab mode" attribute that has values of  "Fixed", "Scrollable" and "Fixed-length" for the various modes in material.io (or something similar to this).